src/SWP/Bundle/ContentBundle/Tests/Controller/RouteControllerTest.php

```diff
@@ -73,7 +71,7 @@ public function testCreateContentRoutesApi()
 
         $this->assertEquals(201, $client->getResponse()->getStatusCode());
         $content = json_decode($client->getResponse()->getContent(), true);
-        $this->assertArraySubset(json_decode('{"id":"\/swp\/default\/routes\/simple-test-route","content":{"id":"\/swp\/default\/content\/test-content-article","title":"Test content article"},"static_prefix":null,"variable_pattern":null,"name":"simple-test-route","children":[],"id_prefix":"\/swp\/default\/routes","template_name":null,"type":"content","_links":{"self":{"href":"\/api\/v1\/content\/routes\/\/simple-test-route"}}, "cache_time_in_seconds":1}', true), $content);
+        $this->assertArraySubset(json_decode('{"id":"\/swp\/123456\/123abc\/routes\/simple-test-route","content":{"id":"\/swp\/123456\/123abc\/content\/test-content-article","title":"Test content article"},"static_prefix":null,"variable_pattern":null,"name":"simple-test-route","children":[],"id_prefix":"\/swp\/123456\/123abc\/routes","template_name":null,"type":"content","_links":{"self":{"href":"\/api\/v1\/content\/routes\/\/simple-test-route"}}, "cache_time_in_seconds":1}', true), $content);
```

Comment:
it's expected value in the test case, not sure if it's so important to make part of it as a const and insert it into the json string. Every expected json string is a bit different, changing it to const is not relavant in this case imho. Whenever you want to change tenant code or an organization code again it can be easily replaced by your editor.